packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -66,8 +66,15 @@ const defaultFilterOptions = createFilterOptions();
 // Number of options to jump in list box when pageup and pagedown keys are used.
 const pageSize = 5;
 
+const defaultIsActiveElementInListbox = (listboxRef) =>
+  listboxRef.current !== null && listboxRef.current.parentElement.contains(document.activeElement);
```

Comment:
Material UI structure is
```js
<Paper>
  <ul>...</ul> // listbox
</Paper>
```

But in Joy UI, there is no parent above listbox so the logic has to be configurable.